README.rst

```diff
@@ -348,120 +356,114 @@ For example:
+The command requires one argument – the path to your application directory
+(that is, to the build source). The default path is ``.`` (the current directory).
 
-This command runs:
+`cartridge build` is executed in two steps:
 
-1. ``cartridge.pre-build`` if the
-   `pre-build file <Special files_>`_ exists.
-   This builds the application in the ``[PATH]`` directory.
-2. ``tarantoolctl rocks make`` if the
-   `rockspec file <Special files_>`_ exists.
-   This installs all Lua rocks to the ``[PATH]`` directory.
+1. If there is a  `pre-build file <Special files_>`_, ``cartridge.pre-build`` builds the application in the ``[PATH]`` directory.
+2. If there is a `rockspec file <Special files_>`_, ``tarantoolctl rocks make`` installs all Lua rocks to the ``[PATH]`` directory.
 
-During step 1 of the ``cartridge build`` command, ``cartridge`` builds the application
-inside the application directory -- unlike when building the application as part
-of the ``cartridge pack`` command, when the application is built in a temporary
-`build directory <Build directory_>`_ and no build artifacts
+First, ``cartridge`` builds the application inside the application directory.
+This is different from ``cartridge pack``, which builds the application inside the
+`build directory <Build directory_>`_. No build artifacts
 remain in the application directory.
 
-During step 2 -- the key step here -- ``cartridge`` installs all dependencies
-specified in the rockspec file (you can find this file within the application
-directory created from template).
-
-(An advanced alternative would be to specify build logic in the
-rockspec as ``cmake`` commands, like we
-`do it <https://github.com/tarantool/cartridge/blob/master/cartridge-scm-1.rockspec#L26>`_
-for ``cartridge``.)
-
-If your application depends on closed-source rocks, or if the build should contain
-rocks from a project added as a submodule, then you need to **install** all these
-dependencies before calling ``tarantoolctl rocks make``.
-You can do it using the file ``cartridge.pre-build`` in your application root
-(again, you can find this file within the application directory created from template).
-In this file, you can specify all rocks to build
+Second, ``cartridge`` installs all dependencies specified in the rockspec file.
+(That file is located in the application directory created from template.)
+
+Alternatively, you can define the build logic in the rockspec in the form of ``cmake`` commands,
+`like we do <https://github.com/tarantool/cartridge/blob/master/cartridge-scm-1.rockspec#L26>`_
+in ``cartridge``.)
+
+If your application depends on closed-source rocks, or if your build contains
+rocks from a project added as a submodule, install all those
+dependencies **before** calling ``tarantoolctl rocks make``.
+You can do so using the file ``cartridge.pre-build`` in your application root
+(again, that file is located in the application directory created from template).
```

Comment:
```suggestion
You can do so using the file ``cartridge.pre-build`` in your application root.
That file is also located in the application directory created from a template.
```

Reply:
I argue the indefinite article is out of place here, as we have been speaking about that application directory and the template it's based upon for a long while already. "From template" is pretty googlable (3 million results). Alternatively, we can say "from your template".